bool 	errorFlag
bool 	stopRequestFlag
uint32 	errorInfo
